packages/yoroi-extension/app/api/ada/lib/storage/bridge/updateTransactions.js

```diff
@@ -3091,16 +3092,16 @@ async function certificateToDb(
   return result;
 }
 
-function compareAndSetIfNewAddressSetHash(id: number, addresses: Array<string>): boolean {
+async function compareAndSetIfNewAddressSetHash(id: number, addresses: Array<string>): Promise<boolean> {
   const requestAddresseSet = new Set(addresses);
   const requestAddresseHash = ObjectHash.sha1(requestAddresseSet);
   const localStorageKey = `TMP/UTXO_REQUEST_ADDRESSES_HASH/${id}`;
-  const prevRequestAddressHash = localStorage.getItem(localStorageKey);
+  const prevRequestAddressHash = await getLocalItem(localStorageKey);
   const isUpdating = prevRequestAddressHash == null || prevRequestAddressHash !== requestAddresseHash;
   if (isUpdating) {
     // <TODO:REMOVE_AFTER_YOROI_LIB_UPGRADE>
```

Comment:
@vsubhuman could you please review this comment? I forgot why it is there.

Reply:
ah, this is because utxo api in the yoroi-lib needs fixing for when the list of addresses got updated. We need to leave this be for now